spec/integration/put_content/new_edition_spec.rb

```diff
@@ -0,0 +1,68 @@
+require "rails_helper"
+
+RSpec.describe "PUT /v2/content when the payload is for a brand new edition" do
+  include PutContentIntegrationHelper
+
+  let(:payload) { default_payload }
+
+  it "creates an edition" do
+    put "/v2/content/#{content_id}", params: payload.to_json
+
+    edition = Edition.last
```

Comment:
I wonder if we can use `subject` somehow to avoid having to put `Edition.last` in all the examples?